api/src/main/java/com/basistech/util/TextDomain.java

```diff
@@ -32,29 +33,33 @@
     /**
      * Create a TextDomain object.
      * 
-     * @param script the ISO 15924 numeric script id
-     * @param language the ISO 639 numeric langauge id
+     * @param script the ISO 15924 code
```

Comment:
What about the set methods?  They will will set a null if passed, not the new defaults.